src/Core/Silk.NET.SilkTouch/NativeApiGenerator.cs

```diff
@@ -157,11 +166,11 @@ INamedTypeSymbol sharedClassSymbol
             
             bool compactFileFormat;
 
-            #if DEBUG
+#if DEBUG
```

Comment:
Does this branch also get hit when `DEBUGANALYZER` is on? i.e. is `DEBUG` a superclass of `DEBUGANALYZER`?